src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -381,16 +447,38 @@ const JupyterFrameManager = ({ onClose, frameRef }) => {
   return null
 }
 
-const NotebookEditorFrame = ({ mode, notebookName, workspace: { workspace: { namespace, name, bucketName } }, cluster: { clusterName, clusterUrl, status } }) => {
+const NotebookEditorFrame = ({ mode, notebookName, workspace: { workspace: { namespace, name, bucketName } }, cluster: { clusterName, clusterUrl, status, labels } }) => {
   console.assert(status === 'Running', 'Expected notebook runtime to be running')
   const frameRef = useRef()
+  const signal = useCancellation()
```

Comment:
It's surprising that we'd want cancellation for one of these paths but not the other. What's the background here?

Reply:
Since all the new notebook setup calls are POSTs, it seemed best to disable cancellation. For the old route though, I just kept what Terra was doing before, which was with cancellation. Should I remove it from the old setup calls as well?